maps/map02.txt

```diff
@@ -1,12 +1,8 @@
-#### ####
-#.     .#
-# ## ## #
-#       #
-# ## ## #
-#       #
-# ## ## #
-    o    
-# ## ## #
-#       #
-#g      #
-#### ####
+##############
+#...#...#....#
+#.#.#.#.#..#.#
+#o#.#.#.#..#.#
+# #.#.#.#..#.#
+# #.#.#.#..#.#
+#g#...#....#.#
+##############
```

Comment:
I like how this level look pretty easy at first glance. And then this wide corridor comes. :)